libcxx/include/__iterator/wrap_iter.h

```diff
@@ -10,6 +10,7 @@
 #ifndef _LIBCPP___ITERATOR_WRAP_ITER_H
 #define _LIBCPP___ITERATOR_WRAP_ITER_H
 
+#include <__compare/ordering.h>
```

Comment:
IIUC we also need to change `__bounded_iter` (provided in `<__iterator/bounded_iter.h>`) which is conditionally used by `span` and `basic_string_view`.
[N4868 [string.view.iterators]/2](https://timsong-cpp.github.io/cppwp/n4868/string.view.iterators#2) and [[span.iterators]/2](https://timsong-cpp.github.io/cppwp/n4868/span.iterators#2) require their iterator types to meet the requirements on container iterators.

Reply:
Thanks for catching this!